Makefile

```diff
@@ -0,0 +1,16 @@
+develop:
+	python setup.py develop
+
+undevelop:
+	python setup.py develop --uninstall
+
+test:
+	flake8 rds-create-freestoragespace-alarms
```

Comment:
I don't know, I have no time to dig, but you certainly don't want to keep something if you don't understand why it's working.